book/setup.md

```diff
@@ -147,19 +209,44 @@ brev login
 
 :::
 
-#### Provisioning environments
+#### Prepare an NVIDIA Brev instance
+
+Later on in the [SciPy 2025 tutorial](https://cfp.scipy.org/scipy2025/talk/GDN8PN/), we'll use a coupon code to provision a new Brev GPU instance environment.
 
-Later on in the [SciPy 2025 tutorial](https://cfp.scipy.org/scipy2025/talk/GDN8PN/), we'll use a coupon code to provision a [new Brev GPU instance environment](https://brev.nvidia.com/environment/new).
-The particular configuration for the code is:
+The particular configuration we'll be using is:
 * 1x NVIDIA L4 GPU
 * 24GiB VRAM
 * 16GiB Ram x 4 CPUS
 * GCP
 
-[![brev-new-environment](images/brev-new-environment-view.png)](https://brev.nvidia.com/environment/new)
+Select it from the [Brev new environment page](https://brev.nvidia.com/environment/new) **OR** run the following command to create a new instance with the same configuration:
```

Comment:
```suggestion
You can select it from the [Brev new environment page](https://brev.nvidia.com/environment/new)

[![brev-new-environment](./images/brev-new-environment-view.png)](https://brev.nvidia.com/environment/new)

**OR** run the following command to create a new instance with the same configuration:
```

As is the selection view image is now out of the "Prepare an NVIDIA Brev instance" subsection and appears in the subsection below that. So I'd suggest maybe placing the image immediately after the web view selection and also removing the 

```
[![brev-new-environment](./images/brev-new-environment-view.png)](https://brev.nvidia.com/environment/new)
```

from below.